RubeesEat.IntegrationTests/SeleniumTests/SplitBillSeleniumTests.cs

```diff
@@ -98,5 +98,20 @@ public void WrongAmountPerPerson()
         
         Assert.That(page.GetErrorMessage(), Is.EqualTo("The sum of individual amounts doesn't match the total bill amount."));
     }
+
+    [Test]
+    public void AddAndRemovePerson()
+    {
+        var page = Start<SplitBillPageObject>();
+        
+        var person = page.ClickAddPerson();
+        
+        Assert.That(page.GetPersonAmountAndNames().Length, Is.EqualTo(1));
+        
+        person.ClickRemoveButton();
+        
+        Assert.That(page.GetPersonAmountAndNames().Length, Is.EqualTo(0));
```

Comment:
I think it would be important here to assert that the person is now available again in the list of people to choose. So we should assert that when we add a person it is no longer in the list. And if we remove it, it is back in the list of items to choose from.